CMSgov/managed-care-review
# Add initial email settings endpoint and return config

## Summary
Details of investigation posted on the bug ticket. 

TL;DR not sure what's going on, add email settings endpoint and more errors to apollo_gql load to try and pinpoint where we are losing these variables.

Other changes
- pulled out shared helper for validating stringlist variables from parameter store and added unit tests
-  mass renaming of an email config variable that was still had an old name and was confusing me. 
- remove rates review parameter store helper function - dead code this variable no longer used 

#### Related issues
https://qmacbis.atlassian.net/browse/MR-3115

#### Test cases covered
- new tests for `validateAndReturnValueArray`

## QA guidance
- Emails on test box still work as expected. You only need to trigger one contract and rates new submission to check this.
- Go to graphql explorer and hit the fetchEmailSettings endpoint
